src/TestSuite/TestCase.php

```diff
@@ -898,6 +898,24 @@ public static function setAppNamespace(string $appNamespace = 'TestApp'): ?strin
         return $previous;
     }
 
+    /**
+     * Adds a fixture to this test case.
+     *
+     * Examples:
+     * - core.Tags
+     * - app.MyRecords
+     * - plugin.MyPluginName.MyModelName
```

Comment:
Should we give some guidance on how this method is to be used? For example 'Use this method inside your test cases' `getFixtures()` method to build up the fixture list'

Reply:
Sounds good. Here on in docs?

Reply:
I added the note in the docblock.